source/faq.txt

```diff
@@ -41,6 +41,25 @@ For applications that require a mix of the new and legacy APIs, ``com.mongodb.Mo
 - Configuration with ``MongoClientSettings`` and ``ConnectionString``, the only difference being that you create instances via constructors instead of a factory class.
 - CRUD API using ``MongoDatabase``, and from there, ``MongoCollection``.  You can access this API via the ``getDatabase()`` method.
 
+How do I use the ``MongoClientSettings`` class?
```

Comment:
comment and suggestion: The answer below seems to answer the question "What is the `MongoClientSettings ` class?" more so than the current question.

 ```suggestion
What is the ``MongoClientSettings`` class?
```

Reply:
Thanks for the note. I agree that before the other edits in this review were performed this edit would have improved the page, [but after the other edits were performed](https://github.com/mongodb/docs-java/commit/295ead332d281e071291e3725a0c5726f68a6706) I think "How do I use..." is more appropriate (content was made more task oriented). 

Let me know if you disagree with this assessment.